master/internal/db/postgres_rp_workspace_bindings_intg_test.go

```diff
@@ -38,14 +56,32 @@ func TestListAllBindings(t *testing.T) {
 }
 
 func TestOverwriteBindings(t *testing.T) {
+	ctx := context.TODO()
+	// Add bindings
+	workspaceIds := []int32{1}
+	poolName := "pool1"
+	err := AddRPWorkspaceBindings(ctx, workspaceIds, poolName)
+	require.NoError(t, err, "failed to add bindings")
```

Comment:
we can talk more on the call, but we should add more checks like checking that the workspaceID is the same and RP name is the same and the number of rows in query result is the same